src/pages/index.js

```diff
@@ -30,55 +36,198 @@ const addCardPopup = new PopupWithForm(
   "#profile__add-card-modal",
   handleAddCardSubmit
 );
+const editAvatarPopup = new PopupWithForm(
+  "#profile__avatar-modal",
+  handleAvatarSubmit
+);
+profileEditPopup.setEventListeners();
+addCardPopup.setEventListeners();
+editAvatarPopup.setEventListeners();
+
 const previewImagePopup = new PopupWithImage("#card_modal");
+const trashConfirmPopup = new PopupWithConfirm("#trashcan-modal", handleDelete);
 
 // Section
-const cardSection = new Section(
-  { items: initialCards, renderer },
-  ".cards__list"
-);
-cardSection.renderItems();
+const cardSection = new Section({ items: [], renderer }, ".cards__list");
 
-// Form validators
+// Validators
+const profileEditForm = document.querySelector("#profileEditForm");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms